### PR TITLE
Feature/circle

### PR DESCRIPTION
**Pull request by fivetran-kristin**
Updates package to latest standards, including:
- Circle CI set up
- model naming conventions
- add passthrough columns

### CODE REVIEW

File: integration_tests/dbt_project.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+name: 'salesforce_integration_tests'
+version: '1.0.0'
```
Comment (fivetran-jamie) [inline integration_tests/dbt_project.yml:2]: should this be 0.1.0?

Comment (fivetran-jamie) [reply integration_tests/dbt_project.yml:2]: everything else looks goof to me!

Comment (fivetran-kristin) [reply integration_tests/dbt_project.yml]: Just updated, thanks! Can you approve?

Comment (fivetran-kristin) [reply integration_tests/dbt_project.yml]: (update to 0.3.0, which will be the next release of this package)

**Review by fivetran-jamie — APPROVED** [review]

**Pull request merged**